cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -555,8 +556,21 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
         Map<String, Object> optimizationProposal = new HashMap<>();
         optimizationProposal.put(CruiseControlRebalanceKeys.SUMMARY.getKey(),
                 proposalJson.getJsonObject(CruiseControlRebalanceKeys.SUMMARY.getKey()).getMap());
-        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
-
+//        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
+
+        String mapAsString = beforeAndAfterBrokerLoad.keySet().stream()
+                .map(key -> key + "\n" + converttoformat(beforeAndAfterBrokerLoad.get(key)) + "\n")
+                .collect(Collectors.joining(", ", "{", "}"));
+        ConfigMap rebalanceMap = new ConfigMapBuilder()
+                .withApiVersion("v1")
+                .withNewMetadata()
+                .withName("kafka-rebalance-before-after-load")
```

Comment:
The name of the config map should include the name of the KafkaRebalance instance to show it is linked.